.circleci/workflows/release-publish.yaml

```diff
@@ -1,91 +1,59 @@
+!yamlscript/v0/
+
+# Define 2 default mappings for clean reuse:
+default-filters =::
+  branches:
+    ignore: /.*/
+  tags:
+    only: /^v\d+\.\d+\.\d+-?.*/
+
+# XXX This might not be required to differ from default-filters
+default-filters-2 =::
+  branches:
+    ignore: /.*/
+  tags:
+    only: /^v\d+\.\d+\.\d+$/
```

Comment:
I think the names of these filters could be `prerelease-filter` and `release-filter`.

Reply:
See f694be9e2
I made comments on the prerelease regex.
I won't change the regex for this PR, since it's a refactoring and should cause no normative changes.

Reply:
Yeah, I think I may have just wanted to hint at the intention. Seems right to make it actually stop malformed release tags like you suggest. But later, of course.

The naming could probably be clearer still. Like `include-prerelease-tags` and `only-release-tags`, or something. But we can do that later as well.